packit_service/worker/helpers/testing_farm.py

```diff
@@ -220,6 +297,17 @@ def _payload(
         else:
             build_log_url = nvr = srpm_url = None
 
+        packit_copr_rpms = (
+            [
+                package
+                for artifact in artifacts
+                if artifact.get("packages")
```

Comment:
I'd be careful, cause this doesn't necessarily mean they are Copr builds. You can also have Koji or URL to RPM repo.

```suggestion
                if artifact.get("type") == "fedora-copr-build" and artifact.get("packages")
```

_Edit_: not valid anymore, since only they get generated only on our side